cassandra-diagnostics-commons/src/main/java/io/smartcat/cassandra/diagnostics/measurement/Measurement.java

```diff
@@ -0,0 +1,157 @@
+package io.smartcat.cassandra.diagnostics.measurement;
+
+import java.util.HashMap;
+import java.util.Map;
+
+/**
+ * This class represents a module generated measurement.
+ * There are two types of measurements:
+ * * Simple measurement: Usually scalar values defining a single measurement
+ * * Complex measurement: Provides rich data for a single point in time
+ */
+public final class Measurement {
+
+    /**
+     * Defines possible measurement types.
+     */
+    public enum MeasurementType {
+        /**
+         * Simple measurement holds timestamp and value.
+         */
+        SIMPLE,
+
+        /**
+         * Complex measurement is a group of measurements which are in fields and have names and values.
+         */
+        COMPLEX
+    }
+
+    /**
+     * Measurements name.
+     */
+    public final String name;
+
+    /**
+     * Measurements type.
+     */
+    public final MeasurementType type;
+
+    /**
+     * Measurement value.
+     */
+    public final double value;
+
+    /**
+     * Measurement time in milliseconds since epoch.
+     */
+    public final long time;
+
+    /**
+     * Measurement tags.
+     */
+    public final Map<String, String> tags;
```

Comment:
Shouldn't it be better to have Map<String, Object> tags instead of Map<String, String>?